### PR TITLE
Dev blowymind99 skills

### PR DESCRIPTION
**Pull request by blowymind99**

### CODE REVIEW

**Review by andreimaff — APPROVED** [review]
nice 💯

File: src/_component/skills/skills.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+.skills__count,
+.skills__icon {
+    padding: 10px 12px 8px 10px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}   
+
+.skills__icon,
+// .skills__count,
```
Comment (tandaraelena) [inline src/_component/skills/skills.scss:34]: Can this comment be deleted?

File: src/_component/skills/skills.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+.skills__count {
+    color: #666666 ;
+    font-size: 18px;
+}
+
+
+.skills_skill--heart { 
+    background-color: #D52027;
+    ;
```
Comment (tandaraelena) [inline src/_component/skills/skills.scss:52]: Probably you forgot to delete this ";" ?

**Review by tandaraelena — APPROVED** [review]
Left 2 minor questions but approved otherwise 👍🏼

**Review by RalucaRbc — APPROVED** [review]
Great, keep it up 🚀

File: src/_component/skills/skills.scss (hunk 2 above)
Comment (blowymind99) [reply src/_component/skills/skills.scss:52]: yes, you are right :D tnx

**Review by BogdanNegura — APPROVED** [review]
👍

**Pull request merged**